_includes/header.html

```diff
@@ -32,6 +32,7 @@
                     <li class="nav-item {% if page.url == "/releases.html" %}active{% endif %}"> <a class="nav-link" href="{{'/releases.html' | relative_url}}">Release Notes</a> </li>
                     <li class="nav-item {% if page.url == "/downloads.html" %}active{% endif %}"> <a class="nav-link" href="{{'/downloads.html' | relative_url}}">Downloads</a> </li>
                     <li class="nav-item {% if page.url == "/contribute.html" %}active{% endif %}"> <a class="nav-link" href="{{'/contribute.html' | relative_url}}">Contribute</a> </li>
+                    <li class="nav-item {% if page.url == "/history/index.html" %}active{% endif %}"> <a class="nav-link" href="{{'/history/index.html' | relative_url}}">History</a> </li>
```

Comment:
Looks like this does not work, the "History" text is not set to "active" after the history page was clicked. I don't understand why and hope that it is only a problem with my local setup.